tests/build/jvm-build.go

```diff
@@ -46,6 +46,7 @@ var _ = framework.JVMBuildSuiteDescribe("JVM Build Service E2E tests", Label("jv
 			Expect(f.AsKubeAdmin.HasController.DeleteApplication(applicationName, testNamespace, false)).To(Succeed())
 			Expect(f.AsKubeAdmin.TektonController.DeleteAllPipelineRunsInASpecificNamespace(testNamespace)).To(Succeed())
 			Expect(f.SandboxController.DeleteUserSignup(f.UserName)).NotTo(BeFalse())
+			Expect(f.AsKubeAdmin.JvmbuildserviceController.DeleteJbsConfig(constants.JBSConfigName, testNamespace)).To(Succeed())
```

Comment:
what's the motivation for deleting the JBSconfig @stuartwdouglas?

should we delete it in rhtap-demo test as well?

Reply:
It should trigger a clean up of the image repo that was created. It should probably be deleted in both places.